escalation/jira-listener/pom.xml

```diff
@@ -15,7 +15,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
-    <quarkus.platform.version>3.2.9.Final</quarkus.platform.version>
+    <quarkus.platform.version>3.12.2</quarkus.platform.version>
```

Comment:
Should the same be done to the software template?

Reply:
no, this is only a bump for side applications (i.e: jira listener and m2k-kfunc), not workflows
I do not remember having seen side applications in the template, but I'll check

Reply:
there aren't side application. In that case - this PR is good to be merged.